packages/sources/s3reader/test-payload.json

```diff
@@ -0,0 +1,6 @@
+{
+ "requests": [{
```

Comment:
Is this a valid test-payload.json for this EA?

Reply:
Added this to the soaktest blacklist so this file is not used for now.
Reason being we need some policies setup for AWS that make it not possible (again for now)